### DISCUSSION TITLE
为什么选择本地代理的存储，在自己电脑上搭建时走的是云盘的地址，在vps上搭建就是走的vps

### DISCUSSION THREAD

**Discussion by fxxisme** [AlistGo/alist#6947] [Q&A] [answered]
夸克云盘和迅雷云盘选择的本地代理，因为vps带宽低了，观看效果很差，在自己电脑上搭建后我看到地址是连的云盘的地址，速度很快，为什么会有区别

**Comment by Simonzhang8** [comment, marked as answer]
本机设备搭建和302就几乎没有区别了，因为从本机转发本机的带宽限制几乎可以认为没有

vps搭建就得通过vps中转文件，会受vps的带宽限制和本机的网速限制
